src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (sourceString) => {
+  const splittedSource = sourceString.split(';');
+  const resultObj = {};
+
+  for (const element of splittedSource) {
+    const resultArray = element.replace(/[[\n\t]/g, '').split(':');
+
+    if (resultArray.length === 2) {
+      resultObj[resultArray[0].trim()] = resultArray[1]
+        .replace(/[',']/g, ',\n')
+        .trim();
```

Comment:
The 'replace' method here is unnecessary and could be removed. It seems to be intended to format the output, but the task requires returning an object, not a string, so formatting is not needed.